### PR TITLE
created the init.sql file

### PR DESCRIPTION
**Pull request by saherhanif**

### CODE REVIEW

**Review by MuhammadAw90 — DISMISSED** [review]
I suggest to create new table for credentials for Employees that have the permission to login to the system.

File: database/init.sql [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+INSERT INTO iteration (iteration_name, project_id, index, iteration_start_date, iteration_end_date) VALUES ('iteration-27',1,'1','2021-08-17','2021-09-06'),('iteration-27',1,'2','2021-09-07','2021-09-13'),('iteration-28',1,'1','2021-09-14','2021-10-04');
+
+--milestone
+INSERT INTO milestone (milestone_name, project_id, milestone_date, description) VALUES ('pi3-deployment',1,'2021-08-17','the deployment of the third pi'),('pi3-cut-off',1,'2021-10-04','the cut-off of the third pi');
+
+--scrum
+INSERT INTO scrum (scrum_name, scrum_master_id, project_id) VALUES ('Agno', '69807',1),('Panay', '102954',3);
+
+--employees
+INSERT INTO employee (idNumber, employee_name, email, phone, productivity, site_id, role_id, project_id, scrum_id) VALUES
```
Comment (tawfiknasser) [inline database/init.sql]: separate insert data commands into another script file.

File: database/init.sql [hunk 2]
```diff
@@ -0,0 +1,120 @@
+BEGIN;
+
+DROP TABLE IF EXISTS employee, role, application, site, country, project, iteration, milestone, scrum CASCADE;
+
+CREATE TABLE role (
+   id SERIAL PRIMARY KEY,
```
Comment (tawfiknasser) [inline database/init.sql:6]: Did you take under consideration using UUID instead serial key? and why did you use serial key ?

**Review by LeenMass — DISMISSED** [review]

**Review by tawfiknasser — APPROVED** [review]

**Comment by tawfiknasser** [comment]
@saherhanif you need two approvals

**Review by LeenMass — APPROVED** [review]
you should edit file initSQLBuild.js
there is a double rows

**Comment by tawfiknasser** [comment]
> you should edit file initSQLBuild.js there is a double rows

@LeenMass  could you comment inside the file review ? https://github.com/WebAhead-cs2/caps-look-api/pull/10/files

File: database/init.sql [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+CREATE TABLE iteration (
+   id SERIAL PRIMARY KEY,
+   iteration_name TEXT,
+   project_id INTEGER REFERENCES project(id),
+   index VARCHAR(255)NOT NULL,
+   iteration_start_date DATE,
+   iteration_end_date DATE
+);
+-- "date" this format YYYY-MM-DD
```
Comment (MuhammadAw90) [inline database/init.sql:37]: please remove this comment
or put it in readMe.md file.

**Review by MuhammadAw90 — APPROVED** [review]
Great job

**Review by Karyum — APPROVED** [review]
Didn't review thoroughly since @tawfiknasser reviewed 👍

**Pull request merged**